src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -156,7 +156,29 @@ public void testNoConflictPG() throws IOException {
         //output has both GATK PrintReads and GATK PrintReads.1 in headers
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads"));
         Assert.assertNotNull(SamReaderFactory.makeDefault().open(outFile).getFileHeader().getProgramRecord("GATK PrintReads.1"));
+    }
+
+    @Test
+    public void testPGOnByDefault() throws IOException {
```

Comment:
I don't like how we're embedding tests for general engine features (`PG on by default for all tools`) within the tests for one tool (`PrintReads`). Recommend that you create a separate test class when testing an engine-wide rather than tool-specific feature, like we did with `CRAMSupportIntegrationTest`